src/Microsoft.Azure.WebJobs/FunctionNameAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// This is for providing a custom name to for an SDK function.
+    /// This is not actually consumed by the SDK runtime currently, but rather by function tooling
```

Comment:
Can we remove this description block? I'd rather not have any Azure Functions specific notes in here.